be/src/vec/aggregate_functions/aggregate_function_hll_union_agg.h

```diff
@@ -51,6 +51,9 @@ namespace doris::vectorized {
 struct AggregateFunctionHLLData {
     HyperLogLog dst_hll {};
 
+    AggregateFunctionHLLData() = default;
+    ~AggregateFunctionHLLData() = default;
```

Comment:
In my opinion，this is seems generated by the compiler automatically？
Where does this memory leak come from